cdc/processor/sinkmanager/mem_quota.go

```diff
@@ -197,6 +208,10 @@ func (m *memQuota) clean(tableID model.TableID) uint64 {
 
 // close the mem quota and notify the blocked acquire.
 func (m *memQuota) close() {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	m.tableMemory = make(map[model.TableID][]*memConsumeRecord)
+	m.usedBytes = 0
```

Comment:
Seems both `memQuota.close()` and `memQuota.clean()` for each table are called in SinkManager.Close(), where `memQuota.clean()` is not necessary.